llpc/builder/llpcPipelineState.cpp

```diff
@@ -804,6 +819,51 @@ void PipelineState::ReadUserDataNodes(
     m_userDataNodes = ArrayRef<ResourceNode>(m_allocUserDataNodes.get(), pNextOuterNode);
 }
 
+// =====================================================================================================================
+// Find the resource node for the given set,binding.
+// For nodeType == Unknown, the function finds any node of the given set,binding.
+// For nodeType == Resource, it matches Resource or CombinedTexture.
+// For nodeType == Sampler, it matches Sampler or CombinedTexture.
+// For nodeType == ResourceMappingNodeType::Sampler, it matches DescriptorSampler or DescriptorCombinedTexture.
```

Comment:
The spelling of the enum values here is inconsistent.

Reply:
Thanks; that was a mishap when I squashed commits from various things I was doing at the same time.